Solution1/Solution1.Tests/BL/Services/WeatherServiceTests.cs

```diff
@@ -25,30 +27,34 @@ public class WeatherServiceTests
         private readonly WeatherService _weatherService;
         private readonly Mock<IValidator<ForecastWeatherRequestDTO>> _validator;
         private readonly Mock<IWeatherApiService> _weatherApiServiceMock;
+        private readonly Mock<IWeatherRepository> _weatherRepositoryMock;
+        private readonly Mock<ILogger<WeatherService>> _loggerMock;
         private readonly string cityName = "Minsk";
         private readonly double temp = 11;
         private readonly string comment = Constants.WeatherComments.Fresh;
+        private readonly ValidationResult validResult = new ValidationResult(new List<ValidationFailure>());
```

Comment:
why part of your fields begin from underscore _ , and part not?

Reply:
+